### PR TITLE
Update structure of project

### PR DESCRIPTION
**Pull request by MikhailJunker**
This pull request contain 4 commits, which: 
- Deleted configuration files of programming language and IDE
- Deleted some excesses source files, modify all remaining source files(for more convenient output of the work result) and added comments at code
- Update .gitignore file
- Change folder's structure of project

### CODE REVIEW

File: source/HashTable.cpp [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
 #include "HashTable.h"
 
-
+//default number of groups in table
 const int HashTable::startSize = 16;
 
-
+//constructor without arguments
 HashTable::HashTable()
 {
```
Comment (rrtty0) [inline source/HashTable.cpp]: Please, move the brackets to line with function declaration for all functions

**Comment by rrtty0** [comment]
Good job! I like this changes, but i write some comment about code at file HashTable.cpp, which needs fixing

**Comment by MikhailJunker** [comment]
Done a new commit with the necessary changes

**Comment by rrtty0** [comment]
Ok, thank you for your job! I will merge your changes at my repository.

**Pull request merged**